### PR TITLE
Update spack docs.

### PR DESCRIPTION
**Pull request by koning**

### CODE REVIEW

File: docs/source/spack.rst [hunk 1]
```diff
@@ -14,32 +14,23 @@ Checkout spack
...
-The merlin addition to spack is currently in pull requests, you can clone a 
-branch that has all of the PR changes here:
-
-.. code:: bash
-
-  git clone https://github.com/koning/spack.git
-  cd spack
-  git checkout feature/koning/merlinwf
+  # The merlin spack package is in the develop branch
+  git co develop
```
Comment (lucpeterson) [inline docs/source/spack.rst]: checkout

File: docs/source/spack.rst [hunk 2]
```diff
@@ -14,32 +14,23 @@ Checkout spack
 **************
 
 
-Get the latest version of spack from Github. This is independent from merlin so
-make sure merlin and spack are in separate directories.
+Get the latest version of spack from Github. This is independent from merlin so make sure merlin and spack are in separate directories.
```
Comment (lucpeterson) [inline docs/source/spack.rst]: wrap line

File: docs/source/spack.rst [hunk 3]
```diff
@@ -68,24 +59,17 @@ build python and all python modules merlin needs including numpy.
...
+syntax, this will create an entirely separate build and module.
 
 .. code:: bash
 
     spack install py-merlinwf%gcc@7.1.0
 
 
-A different python version can be specified as part of the package config. To
-build merlin with python-3.6.8 you would type:
+A different python version can be specified as part of the package config. To build merlin with python-3.6.8 you would type:
```
Comment (lucpeterson) [inline docs/source/spack.rst]: line wrap

File: docs/source/spack.rst (hunk 1 above)
Comment (koning) [reply docs/source/spack.rst]: done

File: docs/source/spack.rst (hunk 2 above)
Comment (koning) [reply docs/source/spack.rst]: done

File: docs/source/spack.rst (hunk 3 above)
Comment (koning) [reply docs/source/spack.rst]: done

**Pull request merged**